rktlet/runtime/helpers.go

```diff
@@ -365,6 +366,9 @@ func generateAppSandboxCommand(req *runtimeApi.RunPodSandboxRequest, uuidfile, s
 	// Add hostnetwork
 	if hasHostNetwork(req.GetConfig()) {
 		cmd = append(cmd, "--net=host", "--hosts-entry=host")
+		if hn, err := os.Hostname(); err == nil {
+			cmd = append(cmd, fmt.Sprintf("--hostname=%s", hn))
```

Comment:
In line 329 (same function), we already add `--hostname=`. But I guess it is not possible to have both `req.Config.Hostname != ""` and `hasHostNetwork(req.GetConfig())`, right?

Reply:
That was my thinking. In any case the latter `--hostname=` takes precedence.